examples/batched-solver/batched-solver.cpp

```diff
@@ -0,0 +1,378 @@
+// SPDX-FileCopyrightText: 2017 - 2024 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// @sect3{Include files}
+
+// This is the main ginkgo header file.
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <random>
+#include <string>
+
+
+#include <ginkgo/ginkgo.hpp>
+
+
+// @sect3{Type aliases for convenience}
+// Use some shortcuts. In Ginkgo, vectors are seen as a gko::matrix::Dense
+// with one column/one row. The advantage of this concept is that using
+// multiple vectors is a now a natural extension of adding columns/rows are
+// necessary.
```

Comment:
This comment doesn't fit in this example as you're using `gko::batch::MultiVector` for the vectors